containres/ideaadministrator/app/config_generator.py

```diff
@@ -120,7 +120,7 @@ def get_vpc_cidr_block(self) -> Optional[str]:
         return value
 
     def get_use_vpc_endpoints(self) -> bool:
-        return Utils.get_value_as_bool('use_vpc_endpoints', self.user_values, False)
+        return Utils.get_value_as_bool('use_vpc_endpoints', self.user_values, True)
```

Comment:
As VPC endpoints represents the generally more secure approach for communication between nodes and AWS services, I desire the app to use VPC endpoints by default.

This additionally facilitates local override of these endpoints facilitating local integration testing using localstack